<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshing</class>
 <widget class="QFrame" name="meshing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="Lbl_H1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MESH SETTINGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_MeshType">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Meshing method&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_MeshType">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select type of mesh: in-built mesing / upload mesh files</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Use Hydro-UQ mesher</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Upload mesh files</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Upload solver specific mesh dicts</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_MeshGen">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Solver used to generate mesh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_MeshGen">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select type of mesh: in-built mesing / upload mesh files</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>FLUENT mesh .msh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I-DEAS mesh ANSYS.ans</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CFX mesh .geo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GAMBIT mesh .neu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gmsh file .msh</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Btn_UploadMesh">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Upload the mesh files or mesh dictionaries</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <property name="text">
      <string>
Upload mesh
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_FineH2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fineness of mesh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="HSl_MeshSize">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="toolTip">
      <string>Define the fineness of the mesh</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_MeshFine">
     <item>
      <widget class="QLabel" name="Lbl_FineMesh">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Very coarse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lbl_CoarseMesh">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Very fine</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_H2Regions">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Region refinements&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="Tbl_Regions">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Table of regions where mesh needs to be refined</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Level (1-5)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>P1(x,y,z)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>P2(x,y,z)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_AddRegion">
     <item>
      <widget class="QToolButton" name="Btn_AddRegion">
       <property name="toolTip">
        <string>Add regions for mesh refinement</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Add region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Btn_RemRegion">
       <property name="toolTip">
        <string>Remove regions for mesh refinement</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Remove region</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="VSp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>272</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
